src/fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -52,9 +52,15 @@
     ),
 )
 
-timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path)
+unadjusted_timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path)
+ophys_timestamps = get_timestamps_from_csv(file_path=ophys_timestamp_file_path, adjust_to_zero=False)
+behavior_timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path, adjust_to_zero=False)
+offset = behavior_timestamps[0].replace(tzinfo=None) - ophys_timestamps[0]
+offset_in_seconds = offset.total_seconds()
+adjusted_timestamps = [timestamp + offset_in_seconds for timestamp in unadjusted_timestamps]
```

Comment:
I'd assume it's likely faster to convert this to a numpy array and add the scalar offset to that rather than iterate through